projects/browser-extension/vue-app/.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
+last 2 versions
+not dead
```

Comment:
Is this a legitimate value?

Reply:
Comes from vue-cli install. Not really necessary in the context of a browser-extension, could be just chrome and firefox. But there's more pressing issues to fix first ;)